nixos/roles/statshost/default.nix

```diff
@@ -132,7 +132,7 @@ in
 {
 
   imports = [
-    ./global-metrics.nix
+    ./global-statshost.nix
```

Comment:
We should keep the global-metrics.nix file, I think this change is too confusing if someone is looking for the place to change the globally allowed metrics. Other config that is specific to the central statshost is currently in `default.nix` guarded by `mkIf cfgStatsGlobal.enable`, line 253. Please put changes there. We have to refactor the whole thing later.